src/NsdManager.h

```diff
@@ -62,6 +62,7 @@ class CJNINsdManager : public CJNIBase
 
   void discoverServices(const std::string& serviceType, int protocolType, const CJNINsdManagerDiscoveryListener& listener);
   void registerService(const CJNINsdServiceInfo& serviceInfo, int protocolType, const CJNINsdManagerRegistrationListener& listener);
+  // Deprecated in API level 34
```

Comment:
https://developer.android.com/reference/android/net/nsd/NsdManager#resolveService(android.net.nsd.NsdServiceInfo,%20android.net.nsd.NsdManager.ResolveListener) seems to indicate API 36 deprecates this?

Reply:
Strange, I was already deprecated at level 34 ([API Diff 33-34](https://developer.android.com/sdk/api_diff/34/changes/android.net.nsd.NsdManager#android.net.nsd.NsdManager.resolveService_changed(android.net.nsd.NsdServiceInfo,%20android.net.nsd.NsdManager.ResolveListener)) and [code](https://android.googlesource.com/platform/prebuilts/fullsdk/sources/+/refs/heads/androidx-media-release/android-34/android/net/nsd/NsdManager.java#1268))

Reply:
As it was already deprecated at this level I think we can leave it as it is. The comment is included only as a reminder to proceed with caution.